apps/backend/app/core/database.py

```
@@ -1,86 +1,171 @@
+        },
     )
+    
     _configure_sqlite(engine)
+    _configure_postgresql(engine)
+    
+    # Log slow queries in development
+    if settings.ENV != "production":
+        @event.listens_for(engine, "before_cursor_execute")
+        def _log_query(conn, cursor, statement, parameters, context, executemany):
+            conn.info.setdefault("query_start_time", []).append(asyncio.get_event_loop().time())
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)